terraform/gcp/modules/monitoring/rekor/rekor_alerts.tf

```diff
@@ -100,8 +104,11 @@ resource "google_monitoring_alert_policy" "rekor_k8s_pod_restart_failing_contain
 }
 
 resource "google_monitoring_alert_policy" "rekor_k8s_pod_unschedulable" {
-  # In the absence of data, incident will auto-close in 7 days
+  # adding a dependency on the associated metric means that Terraform will 
+  # always try to   apply changes to the metric before this alert
```

Comment:
nit, there's some extra spacing in the 'to...apply' here. Here and elsewhere.

Reply:
Good catch, just pushed a fix for this.